addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13672,7 +13672,12 @@ msgctxt "#20473"
 msgid "Premiered"
 msgstr ""
 
-#empty strings from id 20474 to 21329
+#. xbmc/playlists/SmartPlayList.cpp
+msgctxt "#20474"
+msgid "Hdr type"
```

Comment:
casing: HDR type

Reply:
also can you add a description similar to some other strings above to assist translators?

Reply:
Done